shapely/tests/geometry/test_equality.py

```diff
@@ -14,6 +18,13 @@ def test_equality(geom):
     assert not (geom != transformed)
 
 
+@pytest.mark.parametrize("geom", all_non_empty_types)
+def test_inequality(geom):
```

Comment:
Could this be combined with the hand-crafted `test_equality_false`, since both test inequality?

e. g., `[(geom, shapely.transform(geom, lambda x: x + 1)) for geom in all_non_empty_types] +` the set of test cases already there?

Reply:
That can indeed be combined!